src/shogun/machine/gp/InferenceMethod.h

```diff
@@ -34,9 +34,17 @@ enum EInferenceType
 	INF_NONE=0,
 	INF_EXACT=10,
 	INF_FITC=20,
+	INF_FITC_REGRESSION=21,
```

Comment:
These are API changes and have to be documented in the NEWS file since serialised versions wont be compatible anymore